### ISSUE TITLE
aux.py filename in developer version causes problem under Windows

### ISSUE THREAD

**Issue by Fetix** [alvarobartt/investpy#182]
Hi there,
investpy is great! =D

Trying to find an ETF and using your workaround from:
https://github.com/alvarobartt/investpy/issues/148
I ran into the same problem as there and couldn't install the developer version.

I discovered that the issue seems to arise when you name a file "aux" under windows as it is reserved for the system. Renaming the file and all imports allowed me to install the developer version correctly. The problem does not apprear for the current master version as the file is called auxiliary.py.

Cheers,
Fetix

**Comment by typhoon71** [comment]
I can confirm this, I installed the dev branch and had to rename the file since it got the wrong name.
Renaming it solved the import issue.
Strange "aux" being a reserved name, I have a bunch of those around...

**Comment by ymyke** [comment]
I can confirm as well. @alvarobartt , can you please fix this soon because it breaks the developer version for all Windows users?

Others, how did you rename the file if you couldn't check out the developer version in the first place? Did you do the checkout on a different architecture?

**Comment by typhoon71** [comment]
I downloaded the zip of the dev branch, unpacked it and then installed it in a venv, using easy_install.
Didn't use git.

**Comment by alvarobartt** [comment]
Hi guys! First of all sorry for taking too long to answer but I am mad busy this days, I will fix it as soon as possible and thank you for reporting it. I also did not know about aux being a Windows reserved name, thanks for explaining it. I will also integrate the CI with Windows since currently tests/ are just being ran on UNIX systems so that I can spot any other error on Windows usage too.

**Comment by alvarobartt** [comment]
Thank you for reporting it and for your patience! It has already been fixed in ac5b05ea4834f87448d3969f6da8ecedda9cebb9, now you should be able to install the developer version on Windows systems just using the following command:

```
pip install git+https://github.com/alvarobartt/investpy.git@developer
```

__Thank you all again! Remember to star the repo if you found it useful and follow me at GitHub so as to get notified of all the updates!__
